### DISCUSSION TITLE
Does it support Stock prices in INR

### DISCUSSION THREAD

**Discussion by pranab-bala** [moneymanagerex/moneymanagerex#6194] [Q&A]
Hello,
I have setup some stocks in stock portfolios. it is in INR with symbol SAIL.NS or SAIL.BO. If select the line and press on "Refresh Stock Price from yahoo" button, it will throw an error "URL using bad/illegal format or missing URL"

**Comment by renato-mmex** [comment]
Which version of MMEX and which operating system are you using.
Are the settings in Options / Other set to https://finance.yahoo.com/echarts?s=%s

**Reply by tactilis** [reply]
@renato-mmex 

The URL in Options | Other you have referred to has nothing to do with refreshing stock prices.

As @n-stein pointed out in #5932:

>This is a common misconception. The URL in the options dialog has nothing to do with share price downloads, it is simply the page navigated to when you right click on a stock and select "Stock Web Page". The URL for share prices is hard coded into the app and can't be modified by end users.
>
>Stock refresh errors were corrected in 1.6.4 for non-GDPR countries. It has subsequently been corrected for GDPR countries, but you need to use the 1.7.0 beta. Take a backup of your database (beta builds may be unstable, although it's in a decent state currently) and try the beta here: https://ci.appveyor.com/project/whalley/moneymanagerex. Note that the beta will update your database version, so you must use the backup if you need to go back to 1.6.4 for some reason.

**Comment by pranab-bala** [comment]
Thanks for the responses.. appreciate it. I was using 1.64 and now updated to 1.7 beta 1. still the issue is the same

**Comment by renato-mmex** [comment]
under Windows 11 and 1.7 Beta 1 it works in Switzerland without problems on my computer.

which operating system are you using and in which country do you use MMEX with which country settings

**Comment by pranab-bala** [comment]
I am using under windows 11 1.7 beta 1 for India currency INR

**Comment by renato-mmex** [comment]
Could you test connecting your PC to a server in the EU via a VPN connection, because I suspect that your IP address is causing the problem.

**Comment by n-stein** [comment]
@pranab-bala please check the latest beta, this has been fixed by resolving issue #6040.
